pkgs/by-name/ah/ahk_x11/package.nix

```
@@ -0,0 +1,99 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, crystal
+, copyDesktopItems
+, linkFarm
+, fetchgit
+
+, gtk3
+, libxkbcommon
+, xorg
+, libnotify
+, gobject-introspection # needed to build gi-crystal
+, openbox
+, xvfb-run
+, at-spi2-atk
+
+, buildDevTarget ? false # the dev version prints debug info
+} :
+
+let
+  pname = "ahk_x11";
+  version = "1.0.4-b76359f"; # 1.0.4 hasn't been released
```

See https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming
```suggestion
  version = "1.0.3-unstable-2024-04-22";
```

---

Any reason to use latest instead of 1.0.3?

---

1.0.3 uses an old version of gi-crystal which generates bad bindings with new libraries. This leads to build failure. I noted this in the comments above `rev`.